gulpfile.js

```diff
@@ -149,6 +149,7 @@ gulp.task('test', gulp.series('test-run', function testVerify() {
 	let failed = false;
 	function onError(error) {
 		failed = true;
+		console.log(error)
```

Comment:
Could you add a semicolon here?

Reply:
I will delete this line, it should not be there